tests/testapp/serializers.py

```diff
@@ -56,3 +56,20 @@ class LibrarySerializer(FilterSerializerByOrgManaged, ValidatedModelSerializer):
     class Meta:
         model = Library
         fields = '__all__'
+
+
+class ShelfSerializerForBook(FilterSerializerByOrgManaged, ValidatedModelSerializer):
+    class Meta:
+        model = Shelf
+        fields = ['name']
+
+
+class BookWithNestedShelfSerializer(
+    FilterSerializerByOrgManaged, ValidatedModelSerializer
+):
+    shelf = ShelfSerializerForBook(read_only=True)
```

Comment:
Hi, @atb00ker I haven't wrote the code to support writable nested serilaizer, this is only `read_only` nested serializer. Hence I wrote the tests for get requests, only.

Reply:
But we need that code for testing, we can't create an endpoint that doesn't work.

Reply:
Done